src/main/java/mate/academy/service/impl/MovieServiceImpl.java

```diff
@@ -26,4 +26,16 @@ public Movie get(Long id) {
     public List<Movie> getAll() {
         return movieDao.getAll();
     }
+
+    @Override
+    public Boolean update(Movie movie) {
```

Comment:
Consider changing the return type of the `update` method to `void` or `Movie` to better reflect the operation's outcome. The current implementation returns a `Boolean` by comparing the updated object with the retrieved object, which may not work as expected due to potential differences in object references or state.